src/Displayable.hs

```diff
@@ -0,0 +1,123 @@
+{-# LANGUAGE TypeSynonymInstances, FlexibleInstances, UndecidableInstances #-}
+
+module Displayable where 
+
+import DisplayTypes 
+import System.Directory 
+import System.FilePath (pathSeparator)
+import Data.Functor.Identity
+import Data.Default.Class
+import Control.Exception
+import System.IO.Unsafe (unsafePerformIO)
+import Frames
+import Graphics.Rendering.Chart.Renderable 
+import Graphics.Rendering.Chart.Backend.Diagrams 
+import Data.Vinyl.TypeLevel (RecAll)
+import Text.Blaze.Html5 (toHtml)
+import Text.Blaze.Html.Renderer.Text (renderHtml)
+import Data.Text.Lazy (toChunks)
+import qualified Data.Vinyl.Functor as V
+import qualified Data.Text as T
+
+-- instance of Displayable of Record from Frames package
+instance (RecAll V.Identity r Show, ColumnHeaders r, AsVinyl r) => Displayable (Record r) where
```

Comment:
This should be in another file, so we can import Displayable from there, and users can do the same.